crates/flowctl/src/lib.rs

```diff
@@ -211,6 +215,26 @@ where
     }
 }
 
+/// Execute a [`postgrest::Builder`] request returning multiple rows. Unlike [`api_exec`]
+/// which is limited to however many rows Postgrest is configured to return in a single response,
+/// this will issue as many paginated requests as neccesary to fetch every row.
+async fn api_exec_paginated<T>(b: postgrest::Builder) -> anyhow::Result<Vec<T>>
+where
+    for<'de> T: serde::Deserialize<'de> + Send + Sync,
```

Comment:
optional nit: The trait [DeserializeOwned](https://docs.rs/serde/latest/serde/de/trait.DeserializeOwned.html) exists as a more readable alternative to higher ranked trait bounds.